src/widget/wmainmenubar.cpp

```diff
@@ -265,7 +265,15 @@ void WMainMenuBar::initialize() {
     QString fullScreenTitle = tr("&Full Screen");
     QString fullScreenText = tr("Display Mixxx using the full screen");
     auto pViewFullScreen = new QAction(fullScreenTitle, this);
-    QList<QKeySequence> shortcuts{QKeySequence("F11")};
+    QList<QKeySequence> shortcuts;
+#ifdef __APPLE__
+    // On macOS, use only the default system shortcut because there might be
+    // issues with getting F11 working on newer macOS versions:
+    // https://github.com/mixxxdj/mixxx/pull/3011#issuecomment-678678328
+    shortcuts << QKeySequence::FullScreen;
+#else
+    // On Linux and Windows we use F11 and the OS shortcut.
+    shortcuts << QKeySequence("F11");
```

Comment:
Why not wrap only this single line with `#ifndef __APPLE__` to exclude the F11 shortcut?

Reply:
right, #ifndef
thx!